.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Thank you for opening a Pull Request!
+
+Please provide answers to the following questions, delete examples.
+
+--------------------------------------------------------------------------------
+
+## Does this PR concern a dataset or tutorial?
```

Comment:
doesn't this follow from the next question?

Reply:
Yes, in some cases it may, but I've seen PRs where people do not leave much of a description to glean this info from. I'd rather have it be explicit. Even if all they provide is the contribution type, at least I can know who to assign to without having to look at the files.

Leaving it as it for now.